src/api/index.ts

```diff
@@ -170,7 +163,26 @@ export class GithubDeploymentsApiClient implements GithubDeploymentsApi {
       return formattedStatues;
     };
 
-    const formatted = formatStatuses(statuses);
-    return formatted;
+    const page = 100;
+    const pages = Math.ceil(params.deploymentNodeIds.length / page);
+    const statusRequests = [];
+    const formattedStatues: RestDeploymentStatus[] = [];
```

Comment:
formatted "Statues"? 🗽

Reply:
Other than that typo, I'm fine with this PR.

I wonder when GitHub imposed the limit. I'm quite sure I had tried that GraphQL query with many hundreds of nodes before you first released this plugin.

Reply:
Fine. I'll spell it ~~write~~ right. 🙁

Reply:
I think it's statue free now.